app/models/suggestion.rb

```diff
@@ -0,0 +1,8 @@
+class Suggestion < ApplicationRecord
+  belongs_to :user
+  has_many :votes
```

Comment:
Add a presence validation to :user so a suggestion cannot be saved unless it has an associated user